scheduler/src/cook/components.clj

```diff
@@ -61,15 +62,28 @@
                 [:headers "Cache-control"]
                 "max-age=0"))))
 
+(def rate-limiters
+  {:create-job-rate-limiter
+   (fnk [[:settings [:rate-limit expire-minutes job-creation]]]
+     (if job-creation
```

Comment:
Do you want to do `(seq job-creation)` instead?

Reply:
??